package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "jaas-dashboard",
-  "version": "0.1.4",
+  "version": "0.1.6",
```

Comment:
Maybe 5?

Reply:
I was going to go for 9, but yeah 5 is fine :D